Leanplum-iOS-SDK-Example/Classes/AppDelegate.m

```diff
@@ -68,4 +138,56 @@ - (void)configureLeanplum {
     [Leanplum start];
 }
 
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    
+    id notificationCenterClass = NSClassFromString(@"UNUserNotificationCenter");
+    [self configureLeanplum];
+    if (notificationCenterClass) {
```

Comment:
Let's move all of this into a method called "registerForPushNotifications" so that the main method can be readable